files/default/neutron-ha-tool.py

```diff
@@ -233,6 +236,10 @@ def run(args):
     else:
         raise ValueError('Invalid agent_selection_mode')
 
+    if args.target_agent is not None:
```

Comment:
You are right, I think we can assume that having a falsey value is an error here, so we can get rid of the ugly `is not` part.